src/MCPServer/install/README.md

```diff
@@ -1,13 +1,135 @@
+    - True (enabled):  `"1"`, `"yes"`, `"true"` or `"on"`
+    - False (disabled): `"0"`, `"no"`, `"false"` or `"off"`
+
+Certain environment strings are mandatory, i.e. they don't have defaults and
+the application will refuse to start if the user does not give them one.
+
+Please be aware that Archivematica supports different types of distributions
+(Ubuntu/CentOS packages, Ansible or Docker images) and they may override some
+of these settings or provide values to mandatory fields.
+
+This is the full list of environment strings supported:
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_DJANGO_SECRET_KEY`**:
+    - **Description:** a secret key used for cryptographic signing. See [SECRET_KEY](https://docs.djangoproject.com/en/1.8/ref/settings/#secret-key) for more details.
+    - **Type:** `string`
+    - :red_circle: **Mandatory!**
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_SHAREDDIRECTORY`**:
+    - **Description:** location of the Archivematica Shared Directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROCESSINGXMLFILE`**:
+    - **Description:** name of the processing configuration file.
+    - **Type:** `string`
+    - **Default:** `"processingMCP.xml"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_MCPARCHIVEMATICASERVER`**:
+    - **Description:** address of the Gearman server.
+    - **Type:** `string`
+    - **Default:** `"localhost:4730"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCHDIRECTORYPATH`**:
+    - **Description:** location of the Archivematica Watched Directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/watchedDirectories/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROCESSINGDIRECTORY`**:
+    - **Description:** loation of the processing directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/currentlyProcessing/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_REJECTEDDIRECTORY`**:
+    - **Description:** location of the rejected directory.
+    - **Type:** `string`
+    - **Default:** `"%%sharedPath%%rejected/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WAITONAUTOAPPROVE`**:
+    - **Description:** suspend execution of the current thread before attempting to approve a chain when it is pre-configured.
+    - **Type:** `int`
+    - **Default:** `"0"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCHDIRECTORIESPOLLINTERVAL`**:
+    - **Description:** time in seconds between filesystem poll intervals .
+    - **Type:** `int`
+    - **Default:** `"1"`
+
+- **`ARCHIVEMATICA_MCPSERVER_PROTOCOL_LIMITTASKTHREADS`**:
+    - **Description:** max. number of threads that MCPServer will run simultaneously.
+    - **Type:** `int`
+    - **Default:** `"75"`
+
+- **`ARCHIVEMATICA_MCPSERVER_PROTOCOL_LIMITTASKTHREADSSLEEP`**:
+    - **Description:** suspend execution of the current thread for the given number of seconds before retrying to start a new thread when the limit is reached.
```

Comment:
This description is confusing to me and could be reworded for clarity. If not entirely reworded, at least "retrying" should be changed to "trying".